src/Checkbox.tsx

```diff
@@ -3,25 +3,34 @@ import * as React from 'react';
 import MuiCheckbox, {
   CheckboxProps as MuiCheckboxProps,
 } from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
 import { FieldProps } from 'formik';
 import { Omit } from './types';
 
 export interface CheckboxProps
   extends FieldProps,
-    Omit<MuiCheckboxProps, 'form'> {}
+    Omit<MuiCheckboxProps, 'form'> {
+  label: string;
```

Comment:
We should probably make this LabelProps: FormControlLabelProps. Otherwise uses won't be able to change anything else about the label

Reply:
Sure, that makes sense.